### ISSUE TITLE
RemoveLogin is not working

### ISSUE THREAD

**Issue by Anderman** [aspnet/Templates#216]
RemoveLogin is not working because loginProvider, providerKey are null

In the view is a multipart identifier used

```
<input asp-for="@account.LoginProvider" type="hidden" />
```

The controller should something like this

```
 public async Task<IActionResult> RemoveLogin(Account account)
        {
            ManageMessageId? message = ManageMessageId.Error;
            var user = await GetCurrentUserAsync();
            if (user != null)
            {
                var result = await _userManager.RemoveLoginAsync(user, account.LoginProvider, account.ProviderKey);
                if (result.Succeeded)
                {
                    await _signInManager.SignInAsync(user, isPersistent: false);
                    message = ManageMessageId.RemoveLoginSuccess;
                }
            }
            return RedirectToAction(nameof(ManageLogins), new { Message = message });
        }
```

And a viewmodel like

```
public class Account
        {
            public string LoginProvider { get; set; }
            public string ProviderKey { get; set; }
        }
```

**Comment by rustd** [comment]
have you tried it and this is broken with beta8?

**Comment by phenning** [comment]
Merged to dev #227
